src/main/java/Duke.java

```diff
@@ -1,10 +1,208 @@
+                        System.out.println("\tTask added:\t" + listTasks.get(jobCount - 1).printDetails());
+                        System.out.println("\tNow you have " + jobCount + " tasks in the list.\n");
+                        System.out.println("____________________________________________________________");
+                    } catch (IndexOutOfBoundsException e) {
+                        System.out.println("____________________________________________________________\n" +
+                                "☹ OOPS!!! The description of a " + splitInputs[0] + " cannot be empty.\n" +
+                                "____________________________________________________________\n");
+                    }
+                }
+                /** To add event to list */
+                else if ((inputCommand.contains("event"))) {
+                    try {
+                        String[] attributeFinder = splitInputs[1].split("/at");
+                        String eventName = attributeFinder[0];
+                        String eventAt = attributeFinder[1];
+                        newTask(new Events(eventName, eventAt));
+                        System.out.println("____________________________________________________________");
+                        System.out.println("\tTask added:\t" + listTasks.get(jobCount - 1).printDetails());
+                        System.out.println("\tNow you have " + jobCount + " tasks in the list.\n");
+                        System.out.println("____________________________________________________________");
+                    } catch (IndexOutOfBoundsException e) {
+                        System.out.println("____________________________________________________________\n" +
+                                "☹ OOPS!!! The description of a " + splitInputs[0] + " cannot be empty.\n" +
+                                "____________________________________________________________\n");
+                    }
+                } else {
+                    try {
+                        gibberishError();
+                    } catch (NoSuchCommandException e) {
+                        System.out.println("____________________________________________________________\n☹ OOPS!!! I'm sorry, but I don't know what that means :-(\n____________________________________________________________");
+                    }
+                }
+            } else if (inputCommand.length() < 4) {
+                try {
+                    gibberishError();
+                } catch (NoSuchCommandException e) {
+                    System.out.println("____________________________________________________________\n☹ OOPS!!! I'm sorry, but I don't know what that means :-(\n____________________________________________________________");
+                }
+            }
+        }
+
+        {
+            String file2="data/duke.txt";
+            String toOut="";
+            try {
+                for(Task i : listTasks)
```

Comment:
i is reserved for index iteration.
Try using a more descriptive variable name
```suggestion
                for(Task task : listTasks)
```